resources/js/components/Board.js

```diff
@@ -125,7 +125,7 @@ export class Board extends React.Component {
         const tileType = tile == 'blank' ? 'blank' : (tile.contents ? 'folder' : 'word');
         const tileText = tile.contents ? tile.name : tile.text;
         return <td style={{ "backgroundColor": `${tile.color}`}}
-            className="default-tile"
+            className={fileType === 'folder' ? 'folder-tile' : 'default-tile'}
```

Comment:
```suggestion
            className={tileType === 'folder' ? 'folder-tile' : 'default-tile'}
```

Reply:
got an error on my end, i think you meant tileType instead of fileType.

Reply:
Followed through on your suggestion.
It works but all the tiles got messed up.
![image](https://user-images.githubusercontent.com/64504438/165895442-a89e299a-a805-4424-8fdc-a0c826d8e272.png)

Reply:
the default-tile css and folder-tile css are different. Maybe the folder-tile css needs to be the same as default-tile, with the only difference being the double border

Reply:
So it would look like this after duping the css.
![image](https://user-images.githubusercontent.com/64504438/165897443-e4441dbc-cc70-4ea0-9adc-6608e0ec9db4.png)

Not sure if I dig the double border. It still seems pretty hard to tell the difference between the folder and tile

Reply:
The border might look better if it were set further inside the tile. I'll look up how to change that real quick

Reply:
looks like the way to do that is to use an outline instead of the border, and set the outline-offset to be a negative value to have it on the inside.

Reply:
Was messing around with the css:
![image](https://user-images.githubusercontent.com/64504438/165898221-3570a1e8-6535-4b5d-9a4e-f317eeae29fd.png)
This definitely looks more obvious but it doesn't look that great lol

Reply:
is the orange part a border? a different color might be better